hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -856,13 +856,9 @@ protected void rollbackFailedWrites(List<String> instantsToRollback, boolean ski
       } else {
         rollback(instant, skipLocking);
       }
-    }
-    // Delete any heartbeat files for already rolled back commits
-    try {
-      HeartbeatUtils.cleanExpiredHeartbeats(this.heartbeatClient.getAllExistingHeartbeatInstants(),
```

Comment:
Did we need to remove these unused function like `getAllExistingHeartbeatInstants` and `cleanExpiredHeartbeats` after this patch?

Reply:
The function is unused and not really useful at all.